commands/deps_check.go

```diff
@@ -26,6 +26,12 @@ import (
 	"github.com/apache/skywalking-eyes/pkg/deps"
 )
 
+var weakCompatible bool
+
+func init() {
+	DepsCheckCommand.PersistentFlags().BoolVarP(&weakCompatible, "weak-compatible", "w", false, "if set, include the weak-compatible list in dependencies checking")
```

Comment:
To me, "include the weak-compatible list in dependencies checking" doesn't indicate the "checking" accepts or rejects the list in dependencies, which might be ambiguous for users, what do you think if we make it more explicit in the description, and add a note, for example:

if set to true, treat the `weak-compatible` licenses as compatible in dependencies check. Note: when set this to true, make sure to manually confirm that you are using the `weak-compatible` licenses under the required conditions.

Reply:
Thanks @kezhenxu94, I updated the description based on your example. CI fixed.